src/quo2/components/colors/color_picker/component_spec.cljs

```diff
@@ -0,0 +1,22 @@
+(ns quo2.components.colors.color-picker.component-spec
+  (:require [quo2.components.colors.color-picker.view :as color-picker]
+            [reagent.core :as reagent]
+            [test-helpers.component :as h]))
+
+(h/describe "color-picker"
+  (h/test "color picker rendered"
+    (h/render [color-picker/view])
+    (-> (h/expect (h/get-by-test-id (str "quo2-color-picker-sky")))
+        (.toBeTruthy)))
+  (h/test "clicks on a color item"
+    (let [event (h/mock-fn)]
+      (h/render [color-picker/view {:on-change #(event)}])
+      (h/fire-event :press (h/get-by-test-id "quo2-color-picker-copper"))
```

Comment:
nit: ideally you can use an accessibility label instead? and the method `get-by-label-text`

Reply:
Based on [new guidelines](https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#accessibility-labels), it seems that accessibility labels are only used for e2e tests and dynamic labels should be avoided. Also, I've noticed that some other component tests are using test-IDs, which is why I opted to use them in this case. Should I switch to accessibility labels instead and is there any specific reason for this?

Reply:
Yes, some tests were implemented before we realised that the guidance from react-native-testing-library is to prefer accessibility labels over test I.ds where possible.